v3/cypress/e2e/smoke/single-codap-smoke.ts

```diff
@@ -10,7 +10,8 @@ import { CfmElements as cfm } from "../../support/elements/cfm"
 import { AxisHelper as ah } from "../../support/helpers/axis-helper"
 
 
-context("codap single smoke test", () => {
+// TODO: Figure out why opening documents from the CFM is broken in this branch
+context.skip("codap single smoke test", () => {
```

Comment:
Is it broken for everyone on this branch or only on your machine? If it's only your machine this should not be included in the PR.

Reply:
@kswenson It's very strange. There's a comment about it [in this Slack thread](https://concord-consortium.slack.com/archives/C035J6RDAK0/p1744921733378039?thread_ts=1744640961.384969&cid=C035J6RDAK0). It's broken when the Cypress tests run both here and on my machine. I skipped the tests for now so it would deploy. Oddly enough, I can open documents on the deployed branch even though I can't in my local instance of CODAP, on this branch or `main`.

Reply:
Turns out the [Cypress test was failing](https://cloud.cypress.io/projects/msrfxa/runs/7095/test-results/111da057-1faa-4a29-ad75-d9d365f4180d?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D&utm_source=github) in a different way here than locally. I believe I've fixed the underlying problem.